doc/sphinx/addendum/nsatz.rst

```diff
@@ -32,6 +32,36 @@ Nsatz: tactics for proving equalities in integral domains
 
    doing automatic introductions.
 
+   `radicalmax`
+     bound when searching for r such that
+     :math:`c (P−Q) r = \sum_{i=1..s} S_i (P i − Q i)`
```

Comment:
The description should add that this argument must be of type `N` (the type of binary natural numbers).